src/main/java/core/basesyntax/service/FruitService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+
+public interface FruitService {
+    void addFruit(Storage storage, Fruit fruit, int amount);
```

Comment:
we should not pass Storage as a parameter or field anywhere in the application - it should be static and accessible